helm/dagster/templates/helpers/instance/_run-launcher.tpl

```diff
@@ -124,11 +124,30 @@ config:
   {{- end }}
 
   {{- if $k8sRunLauncherConfig.schedulerName }}
-  schedulerName: {{ $k8sRunLauncherConfig.schedulerName | quote }}
+  scheduler_name: {{ $k8sRunLauncherConfig.schedulerName | quote }}
```

Comment:
dang this was getting ignored before?

Reply:
i think it was most likely erroring when set